src/main/java/games/strategy/engine/data/annotations/GameProperty.java

```diff
@@ -36,4 +36,17 @@
    * @return true that this property identifies an adder instead of a setter
    */
   boolean adds();
+
+  /**
+   * Indicates the property is virtual and either has no backing field or uses one or more backing fields associated
+   * with other non-virtual properties.
+   *
+   * <p>
+   * A virtual property is typically used for so-called "convenience" properties that are used to initialize multiple
+   * properties at once. Properties of this type are not required to have get, reset, or clear methods.
+   * </p>
+   *
+   * @return {@code true} if the property is virtual; otherwise {@code false}.
+   */
+  boolean virtual() default false;
```

Comment:
This was my creative way to work around validation errors for properties that can be set in the XML but do not have a true backing field because they are typically an alias to conveniently set one or more properties at once.  I'm not sure if calling them "virtual properties" is the best choice for a name.